src/main/java/com/plzgraduate/myongjigraduatebe/graduation/domain/service/major/MandatorySpecialCaseInformation.java

```diff
@@ -0,0 +1,22 @@
+package com.plzgraduate.myongjigraduatebe.graduation.domain.service.major;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@Builder
+@RequiredArgsConstructor
+public class MandatorySpecialCaseInformation {
+
+	private final boolean completeMandatorySpecialCase;
```

Comment:
```suggestion
	private final boolean isCompleteMandatorySpecialCase;
```
조금 더 명확한 변수명으로 요건 어떤가요?

Reply:
[cb787a6](https://github.com/Myongji-Graduate/MyongjiGraduate-BE/pull/271/commits/cb787a63dc36f74b0950cab894c7a22af63abd61)